File: Asocioacion/python/segundoEjercicio/Aprendiz.py
class Aprendiz:
    
    c=0
    
    def __init__(self, nombreAprendiz):
        self.nombreAprendiz=nombreAprendiz
        self.computadoras = []
        self.__class__.c+=1
    
    def getNombreAprendiz(self):
        return self.nombreAprendiz
    
    def setNombreAprendiz(self, nombreAprendiz):
        self.nombreAprendiz = nombreAprendiz
    
    def agregarComputadora(self,computadora):
        self.computadoras.append(computadora)
        
    def verComputadoras(self):
        return self.computadoras

File: Asocioacion/php/segundoEjercicio/Aprendiz.php
<?php

class Aprendiz{

    private string $nombreAprendiz;
    private $computadora = [];

    public function __construct(string $nombreAprendiz){

        $this->nombreAprendiz = $nombreAprendiz;

    }

    //Nombre del aprendiz

    public function getNombreAprendiz(){
        return $this->nombreAprendiz;
    }

    public function setNombreAprendiz($nombreAprendiz){
        $this->nombreAprendiz = $nombreAprendiz;
    }

    //Agregar computadoras

    public function agregarComputadora($computadoras){
        array_push($this->computadora,$computadoras);
    }

    //Ver computadoras

    public function verComputadoras(){
        return $this->computadora;
    }

}

?>